ovos_core/skill_manager.py

```diff
@@ -469,86 +363,29 @@ def _load_plugin_skill(self, skill_id, skill_plugin):
-            LOG.info(f"Please refactor {skill_id} to specify offline network requirements")
-            skill_path = skill_ids.get(skill_id)
-            if skill_path is not None:
-                self._load_skill(skill_path)
-            else:
-                LOG.error(f'Priority skill {skill_id} can\'t be found')
-
     def run(self):
         """Run the skill manager thread."""
-        self.load_priority()
-
         self.status.set_alive()
 
         self._load_on_startup()
 
-        if self.skills_config.get("wait_for_internet", False):
-            LOG.warning("`wait_for_internet` is a deprecated option, update to "
-                        "specify `network_skills` or `internet_skills` in "
-                        "`ready_settings`")
-            # NOTE - self._connected_event will never be set
-            # if PHAL plugin is not running to emit the connected events
-            while not self._connected_event.is_set():
-                # Ensure we don't block here forever if the plugin is not installed
-                self._sync_skill_loading_state()
-                sleep(1)
-            LOG.debug("Internet Connected")
-        else:
-            # trigger a sync so we dont need to wait for the plugin to volunteer info
-            self._sync_skill_loading_state()
+        # trigger a sync so we dont need to wait for the plugin to volunteer info
+        self._sync_skill_loading_state()
 
-        if "network_skills" in self.config.get("ready_settings"):
-            self._network_event.wait()  # Wait for user to connect to network
-            if self._network_loaded.wait(self._network_skill_timeout):
-                LOG.debug("Network skills loaded")
-            else:
-                LOG.error("Gave up waiting for network skills to load")
-        if "internet_skills" in self.config.get("ready_settings"):
-            self._connected_event.wait()  # Wait for user to connect to network
-            if self._internet_loaded.wait(self._network_skill_timeout):
-                LOG.debug("Internet skills loaded")
-            else:
-                LOG.error("Gave up waiting for internet skills to load")
         if not all((self._network_loaded.is_set(),
                     self._internet_loaded.is_set())):
             self.bus.emit(Message(
                 'mycroft.skills.error',
                 {'internet_loaded': self._internet_loaded.is_set(),
                  'network_loaded': self._network_loaded.is_set()}))
+
         self.bus.emit(Message('mycroft.skills.initialized'))
 
         self.status.set_ready()
 
-        if self._gui_event.is_set() and self._connected_event.is_set():
-            LOG.info("Skills all loaded!")
-        elif not self._connected_event.is_set():
-            LOG.info("Offline Skills loaded, waiting for Internet to load more!")
-        elif not self._gui_event.is_set():
-            LOG.info("Skills loaded, waiting for GUI to load more!")
+        self.bus.emit(Message('mycroft.ready'))
```

Comment:
The meaning of this message is changed here; can this emit a new event that is not ambiguous and put `mycroft.ready` behind a config flag to enable a dev to implement backwards-compat. via a plugin?